swagger.yaml

```diff
@@ -1,42 +1,254 @@
 swagger: "2.0"
+
 info:
   title: dp-search-reindex-api
   description: "Provides details about search reindex jobs and enables creation and running of them"
   version: 1.0.0
   license:
     name: "Open Government Licence v3.0"
     url: "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/"
+
 basePath: "/v1"
+
 schemes:
   - http
+
 tags:
-  - name: "hello"
+  - name: "Public"
```

Comment:
This api will never be made public. All endpoints are for internal use only.

So we have `"Private User"` and `"Private"` endpoints only. All the ones currently in the swagger spec should reference the `"Private User"`

Reply:
Future endpoints will be for `"Private"` use only. This means backend applications are the only apps that will make use of these endpoints and not users (e.g. Florence - internal CMS)

Reply:
replaced all instances of "Public" with "Private User' now